react/week4/github-reasecher/src/context/UserSearchResults.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { searchContext } from './createContext'
+
+const UserName = (props) => {
+    return (
+        <div>
+            {props.username}
+        </div>)
+}
+
+const UserSearchResults = () => {
+    const gitUserNames = React.useContext(searchContext)
```

Comment:
Since you are only using "users" from the gitUserNames, you could destructure it instead:
```suggestion
    const { users } = React.useContext(searchContext)
```